src/CMakeLists.txt

```diff
@@ -126,17 +129,20 @@ if (HAVE_GETADDRINFO_A)
 endif (HAVE_GETADDRINFO_A)
 
 
-
 if (UNIX)
 	if (APPLE)
 		set (MOSQ_LIBS ${MOSQ_LIBS} dl m)
-	else (APPLE)
-		set (MOSQ_LIBS ${MOSQ_LIBS} dl m)
-		find_library(LIBRT rt)
-		if (LIBRT)
-			set (MOSQ_LIBS ${MOSQ_LIBS} rt)
-		endif (LIBRT)
-	endif (APPLE)
+    else()
```

Comment:
why not just an elseif instead of nesting further here? would leave the positive logic in place too, if apple, if qnx, else, rather than mixing positive and negative logic.